README.md

```diff
@@ -1,284 +1,126 @@
-| routes `check`          | 🔄                                                                                                                                                |
-| routes `status`         | 🔄                                                                                                                                                |
-| routes `has`            | ✅                                                                                                                                                |
-| routes `missing`        | ✅                                                                                                                                                |
-| routes `locale`         | 🔄                                                                                                                                                |
-| routes `middlewarePath` | ✅                                                                                                                                                |
-| images                  | ❌ (see [Cloudflare's Image Resizing documentation](https://developers.cloudflare.com/images/image-resizing/integration-with-frameworks/#nextjs)) |
-| wildcard                | 🔄                                                                                                                                                |
-| overrides               | 🔄                                                                                                                                                |
-| cache                   | ❌                                                                                                                                                |
-
-- ✅: Supported
-- 🔄: Not currently supported, but it's probably possible and we may add support in the future
-- ❌: Not supported and unlikely we ever will support this
+### 3. Deploy your application to Cloudflare Pages
 
-## Examples
+You can easily deploy to Cloudflare Pages via the [automatic git integration](https://developers.cloudflare.com/pages/platform/git-integration/), to do so, start by committing and pushing your application's code to a Github/GitLab repository.
 
-### [Next.js 13's `app` Directory](https://beta.nextjs.org/docs/routing/fundamentals#the-app-directory)
+Next in the [Cloudflare Dashboard](dash.cloudflare.com) to create a new Pages project:
 
-Add the following to `next.config.js`:
+- Navigate to the project creation pages (_Your account Home_ > _Pages_ > _Create a project_ > _Connect to Git_)
+- Select the Github/GitLab repository you pushed your code to
+- Choose a project name and your production branch
+- Select _Next.js_ as the _Framework preset_
+- Provide the following options:
+  | Option | Value |
+  | ---------------------- | ----------------------------------------------------- |
+  | Build command | `npx @cloudflare/next-on-pages --experimental-minify` |
```

Comment:
Do we want to start suggesting that people pin their `@cloudflare/next-on-pages` version? Or keep it to latest (esp. when in 0.X.X releases)?

Reply:
I am not sure.... 😕

do you think we should pin it? (since we don't pin next itself it kind of feels natural to me also not to pin `@cloudflare/next-on-pages`?)

Reply:
I think we should pin when get to 1.0.0, but probably not bother before that.

Reply:
sure, sounds good to me :slightly_smiling_face::+1: